deployment/refinery-modules/refinery/manifests/docker_.pp

```diff
@@ -0,0 +1,5 @@
+class refinery::docker_ {
+  class { 'docker':
```

Comment:
And yes, it should be changed in the rest of the manifests but in a separate branch.